docs/4.operation/5.rancher.md

```diff
@@ -0,0 +1,22 @@
+## Rancher
+
+FIWARE Ops team is currently deploying a customised version of
+[Rancher 1.6](https://rancher.com/docs/rancher/v1.6/en/) on top of FIWARE Lab.
+This tool allows users with a Cloud account in FIWARE Lab to deploy and
+manage a Docker Swarm cluster on FIWARE Lab. Through which they can
+deploy their FIWARE applications on the Lab using docker containers.
+This is a service offered to the users, and as such FIWARE Lab Node
+Administrators do not have much to do. They only need to be sure that:
+
+-   Their OpenStack APIs are reachable from FIWARE Lab servers (which
+    should be the default case).
+
+-   Provide the configuration of MTU within their node to users
```

Comment:
which MTU? and how to do it?

Reply:
@flopezag The MTU for the nodes Rancher runs on I believe.

Reply:
The question here is to resolve the ambiguity. If we mention that we should configure the MTU inside the nodes, we should describe in short term how we should it and even more important which value we should assign to the MTU.

Reply:
I added some elaboration

Reply:
LGTM